src/test/java/com/puppycrawl/tools/checkstyle/checks/coding/IllegalInstantiationCheckTest.java

```diff
@@ -41,7 +41,8 @@
     @Override
     protected String getPath(String filename) throws IOException {
         return super.getPath("checks" + File.separator
-                + "coding" + File.separator + filename);
+                + "coding" + File.separator
+                + "illegalinstantiation" + File.separator + filename);
```

Comment:
@rnveach I've changed `getNonCompilablePath` and moved resources to new location.